app/views/questionnaires/show.html.erb

```diff
@@ -0,0 +1,294 @@
+            <div class="l-questionnaire__field form__group-item">
+              <%= f.form_row :telephone_number do %>
+                  <%= f.errors_for :telephone_number %>
+                  <%= f.label :telephone_number, t('questionnaire.telephone_number.label'), class: 'form__label-heading' %>
+                  <%= f.text_field :telephone_number %>
+              <% end %>
+            </div>
+          </div>
+        </fieldset>
+
+        <fieldset class="form__group">
+          <legend class="l-questionnaire__legend"><%= t('questionnaire.address.heading') %></legend>
+          <p><%= t('questionnaire.address.description') %></p>
+
+          <div class="l-questionnaire__row">
+            <div class="l-questionnaire__field">
+              <%= f.form_row :address_line_1 do %>
+                  <%= f.errors_for :address_line_1 %>
+                  <%= f.label :address_line_1, t('questionnaire.address.label_one'), class: 'form__label-heading' %>
+                  <%= f.text_field :address_line_1 %>
+              <% end %>
+
+              <%= f.form_row :address_line_2 do %>
+                  <%= f.errors_for :address_line_2 %>
+                  <%= f.label :address_line_2, t('questionnaire.address.label_two'), class: 'form__label-heading' %>
+                  <%= f.text_field :address_line_2 %>
+              <% end %>
+
+              <%= f.form_row :address_town do %>
+                  <%= f.errors_for :address_town %>
+                  <%= f.label :address_town, t('questionnaire.address.label_three'), class: 'form__label-heading' %>
+                  <%= f.text_field :address_town %>
+              <% end %>
+
+              <%= f.form_row :address_county do %>
+                  <%= f.errors_for :address_county %>
+                  <%= f.label :address_county, t('questionnaire.address.label_four'), class: 'form__label-heading' %>
+                  <%= f.text_field :address_county %>
+              <% end %>
+
+              <%= f.form_row :address_postcode do %>
+                  <%= f.errors_for :address_postcode %>
+                  <%= f.label :address_postcode, t('questionnaire.address.label_five'), class: 'form__label-heading' %>
+                  <%= f.text_field :address_postcode %>
+              <% end %>
+            </div>
+          </div>
+        </fieldset>
+      </section>
+
+      <hr>
+
+      <section>
+        <%= heading_tag(t('questionnaire.providing_your_services.heading'), level: 2, class: 'heading-small') %>
+        <p class="l-questionnaire__section-description"><%= t('questionnaire.providing_your_services.description') %></p>
+
+        <fieldset class="form__group">
+          <legend class="l-questionnaire__legend"><%= t('questionnaire.in_person_advice_methods.heading') %></legend>
+
+          <%= f.form_row :in_person_advice_methods do %>
+              <%= f.errors_for :in_person_advice_methods %>
+              <% InPersonAdviceMethod.all.each do |in_person_advice_method| %>
```

Comment:
As @lshepstone said, all of these need to be changed to `collection_check_boxes` and such.